datashare-tasks/src/main/java/org/icij/datashare/asynctasks/CheckedBiFunction.java

```diff
@@ -0,0 +1,6 @@
+package org.icij.datashare.asynctasks;
+
+@FunctionalInterface
+public interface CheckedBiFunction<T, U, R, E extends Exception> {
```

Comment:
question: I did not find where it is in use. I noticed that Bifunction has been removed in tasks, is this one still relevant?